code/lib/preview-api/src/modules/client-api/StoryStoreFacade.ts

```diff
@@ -21,8 +21,11 @@ import { logger } from '@storybook/client-logger';
 import type { StoryStore } from '../../store';
 import { userOrAutoTitle, sortStoriesV6 } from '../../store';
 
+// this is not prefixed with sb because it is user generated
 export const AUTODOCS_TAG = 'autodocs';
-export const STORIES_MDX_TAG = 'stories-mdx';
+export const STORIES_MDX_TAG = 'sb:stories-mdx';
+export const STORY_TAG = 'sb:story';
+export const DOCS_TAG = 'sb:docs';
```

Comment:
If we want to make this manageable, we need to move these tags to a constant that could be shared between other packages. Not sure where